README.md

```diff
@@ -1,4 +1,10 @@
 # lgrenier_project
-BIO 
+BIO
+
 My name is Laurence, I am a master student in psychology at Montreal University.
 I am interested in social neurosciences and cognitive neuroscience. 
+
+<a href="https://github.com/lalou97">
+   <img src="https://avatars.githubusercontent.com/u/87998890?v=4" width="100px;" alt=""/>
```

Comment:
consider updating your avatar on github. This avatar is used in various places, including here, so it is less confusing if you do not use the default. It does not have to be a picture though, lots of people use cartoons and brain images in lieu of an avatar.